setup-linux-bridge.sh

```diff
@@ -34,12 +28,11 @@ if [[ $(nmcli conn | grep -c brcnv) -eq 0 ]]; then
      esac
   done
   echo -e "default dev: $default_device ($profile_name)\nsecondary dev: $secondary_device ($secondary_profile_name)"
-
   mac=$(sudo nmcli -g GENERAL.HWADDR dev show $default_device | sed -e 's/\\//g')
 
 
   nmcli conn add type bridge ifname brcnv con-name brcnv 802-3-ethernet.cloned-mac-address $mac ipv4.method auto ipv4.dhcp-client-id "mac" connection.autoconnect no bridge.stp no
-  nmcli conn add type bond ifname bond0 con-name bond0 bond.options "mode=balance-xor,miimon=100,xmit_hash_policy=5" master brcnv connection.autoconnect no
+  nmcli conn add type bond ifname bond0 con-name bond0 bond.options "mode=balance-alb,tlb_dynamic_lb=0,miimon=100,xmit_hash_policy=5" master brcnv connection.autoconnect no
```

Comment:
I believe you may want to add "primary=$default_device" to the bond.options bit, to ensure the interface with PRIMARY_MAC on it is always the default one when it's up (and also the bond's default MAC).

Reply:
I believe this PR is just to mirror what the customer does now and what was tested on their setup. Would you mind if we followed up with PRIMARY_MAC in a new PR? And if we do, would you help us getting this new change tested on customer's setup? We'd like to keep this repo as close to what is used as possible and as close to what is tested as possible.